Frends.AzureBlobStorage.DownloadBlob/Frends.AzureBlobStorage.DownloadBlob/Definitions/Input.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+
+namespace Frends.AzureBlobStorage.DownloadBlob
+{
+    /// <summary>
+    /// Input-class for DownloadBlob-task.
+    /// </summary>
+    public class Input

```

Comment:
I am unsure of reasons, but I kind of like the previous naming of source and destination. Here we now have Input and Options and it is somewhat confusing, since judging by their contents they are still in fact source and destination. Should we have better naming in those?

Reply:
Renamed to source/destination.